content/en/APIUseCases/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "API Use Cases"
+linkTitle: "API Use Cases"
+weight: 100
+description: >
+  Includes practical uses for our API.
+---
+
+{{% pageinfo %}}
+The pages in this book describe how you can use our API to manage your Assets
```

Comment:
assets

We discussed the capitalization for _assets_ this week.

Reply:
```suggestion
The pages in this book describe how you can use our API to manage your assets
```